presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/metadata/QueryIOMetadata.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.spi.eventlistener.metadata;
+
+import java.util.List;
+import java.util.Optional;
+
+public class QueryIOMetadata
```

Comment:
I'd merge this one into QueryMetadata. I don't think it's worth having an separate class for it.